appveyor.yml

```
@@ -13,6 +13,8 @@ version: "{build}"
 environment:
   # Test against these versions of Node.js
   matrix:
+    - nodejs_version: "14"
```

```suggestion
    - nodejs_version: "14.2.0"
```

It seems this version doesn't have the bug (an access violation in V8 on GC, it looks like).